modules/widgets/milestone/milestone.php

```diff
@@ -205,15 +205,23 @@ function widget( $args, $instance ) {
 			echo $args['before_title'] . $title . $args['after_title'];
 		}
 
-		$data = $this->get_widget_data( $instance );
-
-		self::$config_js['instances'][] = array(
+		$data   = $this->get_widget_data( $instance );
+		$config = array(
 			'id'      => $args['widget_id'],
 			'message' => $data['message'],
-			'refresh' => $data['refresh']
+			'refresh' => $data['refresh'],
 		);
 
-		echo '<div class="milestone-content">';
+		/*
+		 * Sidebars may be configured to not expose the `widget_id`. Example: `twentytwenty` footer areas.
+		 *
+		 * We need our own unique identifier.
+		 */
+		$config['content_id'] = $args['widget_id'] . '-content';
```

Comment:
How about adding this new key directly to the $config array as it is being built above? What do you think?

Reply:
@jeherve I felt we needed a multi-line comment to explain why this value exists. It may be a matter of personal preference, but with that constraint, it feels cleaner to assign to `$config` after its built instead of interrupting the structure with longer comment.

But it's not a strong preference, do you want me to change the implementation?

Reply:
That makes sense. I would have gone with everything inside the main array, but I am okay with your implementation too. Thanks for the explanation!